main.js

```diff
@@ -0,0 +1,308 @@
+        span.innerHTML = '!';
+    } else {
+        span.innerHTML = '';
+    }
+    return span;
+}
+
+
+//4. (continue)...
+// eslint-disable-next-line no-unused-vars
+const addElementToUl = () => {
+    for (let i = 1; i <= productsListToBuy.length; i++) {
+        const li = document.createElement('li');
+        li.innerHTML = `<div class = "list-el"><p>${productsListToBuy[i].productName}.
+            weight: ${productsListToBuy[i].productData.weight}.
+            price: ${productsListToBuy[i].productPrice}.
+            country: ${productsListToBuy[i].productCountry} <img src="${showFlag(productsListToBuy[i])}" alt="Flag">
+            provider: ${productsListToBuy[i].productProvider}
+            sugar free: ${productsListToBuy[i].productData.sugarFree}.
+            date of expiry: ${productsListToBuy[i].productData.dateOfExpiry}.
+            certificate: ${productsListToBuy[i].productData.certificate}.
+            <p></div>`;
+
+    li.appendChild(attract_customer_attention(productsListToBuy[i]));
+    if (i % 2 === 0) {
+        li.style.background = 'lightyellow'
+    } else (li.style.background = 'lightblue')
+    sectionMain_ul.appendChild(li);
+    }
+}
+
+const additionalInfo = document.createElement('div');
+additionalInfo.classList.add('additional-info');
+sectionMain.appendChild(additionalInfo);
+
+// 9. Додати будь-який HTML-тег, в якого записати вартість всього продуктового кошика
+const totalPriceBasket = document.createElement('div');
+totalPriceBasket.classList.add('total-price__food-basket');
+additionalInfo.appendChild(totalPriceBasket);
+
+const showSum = () => {
+    let sum = 0;
+    for (let i = 0; i < productsListToBuy.length; i++) {
+        sum += productsListToBuy[i].productPrice;
+    }
+    return sum.toFixed(2);
+}
+
+const headingBasket = document.createElement('h3');
+headingBasket.innerHTML = 'Basket sum: ' + showSum();
+totalPriceBasket.appendChild(headingBasket);
+
+// 10. Додати будь-який HTML-тег, в якого записати найдорожчий із продуктів цього кошика
+const maxPriceProduct = document.createElement('div');
+maxPriceProduct.classList.add('max-price-product__food-basket');
+additionalInfo.appendChild(maxPriceProduct);
+
+const findMaxPriceProduct = () => {
+    const max = Math.max(...productsListToBuy.map(item => item.productPrice));
+    const mostExpensiveProduct = productsListToBuy.find(item => item.productPrice === max).productName;
+    return `${max} for ${mostExpensiveProduct}`;
+}
+
+const maxPriceHeading = document.createElement('h3');
+maxPriceHeading.innerHTML = `The most expensive product is: ${findMaxPriceProduct()}`;
+maxPriceProduct.appendChild(maxPriceHeading);
+
+// 11. Додати будь-який HTML-тег, в якого записати середню ціну по продуктах
+const averagePrice = document.createElement('div');
+additionalInfo.appendChild(averagePrice);
+
+const findAveragePrice = () => {
+    const averagePrice = Math.round((showSum() / productsListToBuy.length) * 100) /100;
+    return averagePrice;
```

Comment:
It can be simplified 
```suggestion
    return Math.round((showSum() / productsListToBuy.length) * 100) /100;
```